new-lamassu-admin/src/pages/Customers/CustomerData.js

```diff
@@ -75,127 +78,135 @@ const CustomerData = ({ customer, updateCustomer }) => {
+      firstName: Yup.string().required(),
+      lastName: Yup.string().required(),
+      documentNumber: Yup.string().required(),
+      dateOfBirth: Yup.string().required(),
+      gender: Yup.string().required(),
+      country: Yup.string().required(),
+      expirationDate: Yup.string().required()
     }),
     usSsn: Yup.object().shape({
-      usSsn: Yup.string()
+      usSsn: Yup.string().required()
+    }),
+    idCardPhoto: Yup.object().shape({
+      idCardPhoto: Yup.mixed().required()
+    }),
+    frontCamera: Yup.object().shape({
+      frontCamera: Yup.mixed().required()
     })
   }
 
   const idScanElements = [
     {
-      name: 'name',
-      label: 'Name',
-      value: `${getName(customer)}`,
+      name: 'firstName',
+      label: 'First name',
       component: TextInput
     },
     {
-      name: 'idNumber',
+      name: 'documentNumber',
       label: 'ID number',
-      value: R.path(['documentNumber'])(idData) ?? '',
-      component: TextInput
-    },
-    {
-      name: 'birthDate',
-      label: 'Birth Date',
-      value: (rawDob && format('yyyy-MM-dd', rawDob)) ?? '',
       component: TextInput
     },
     {
-      name: 'age',
-      label: 'Age',
-      value: (rawDob && differenceInYears(rawDob, new Date())) ?? '',
+      name: 'dateOfBirth',
+      label: 'Birthdate',
       component: TextInput
     },
     {
       name: 'gender',
       label: 'Gender',
-      value: R.path(['gender'])(idData) ?? '',
       component: TextInput
     },
     {
-      name: 'state',
-      label: country === 'Canada' ? 'Province' : 'State',
-      value: R.path(['state'])(idData) ?? '',
+      name: 'lastName',
+      label: 'Last name',
       component: TextInput
     },
     {
       name: 'expirationDate',
       label: 'Expiration Date',
-      value:
-        (rawExpirationDate && format('yyyy-MM-dd', rawExpirationDate)) ?? '',
+      component: TextInput
+    },
+    {
+      name: 'country',
+      label: 'Country',
       component: TextInput
     }
   ]
 
   const usSsnElements = [
     {
-      name: 'us ssn',
+      name: 'usSsn',
       label: 'US SSN',
-      value: `${customer.usSsn ?? ''}`,
       component: TextInput,
       size: 190
     }
   ]
 
+  const idCardPhotoElements = [{ name: 'idCardPhoto' }]
+  const frontCameraElements = [{ name: 'frontCamera' }]
+
   const initialValues = {
     idScan: {
-      name: '',
-      idNumber: '',
-      birthDate: '',
-      age: '',
-      gender: '',
-      state: '',
-      expirationDate: ''
+      firstName: R.path(['firstName'])(idData) ?? '',
+      lastName: R.path(['lastName'])(idData) ?? '',
+      documentNumber: R.path(['documentNumber'])(idData) ?? '',
+      dateOfBirth: (rawDob && format('yyyy-MM-dd', rawDob)) ?? '',
+      gender: R.path(['gender'])(idData) ?? '',
+      country: R.path(['country'])(idData) ?? '',
+      expirationDate:
+        (rawExpirationDate && format('yyyy-MM-dd', rawExpirationDate)) ?? ''
```

Comment:
`dateOfBirth` and `expirationDate` should still be throwing out errors right? Since the date-fns fix is not yet merged